src/base/bittorrent/session.cpp

```diff
@@ -2473,7 +2479,7 @@ void Session::saveResumeData()
         const std::vector<lt::alert *> alerts = getPendingAlerts(lt::seconds {30});
         if (alerts.empty())
         {
-            LogMsg(tr("Error: Aborted saving resume data for %1 outstanding torrents.").arg(QString::number(m_numResumeData))
+            LogMsg(tr("Save resume data aborted. Number of outstanding torrents: %1").arg(QString::number(m_numResumeData))
```

Comment:
>Save resume data aborted

Doesn't seem right. I'd keep the existing sentence: "Aborted saving resume data"